pkg/operator/ceph/nvmeof_recoverer/clustermanager/handler.go

```diff
@@ -0,0 +1,34 @@
+package clustermanager
+
+import (
+	"github.com/coreos/pkg/capnslog"
+)
+
+var logger = capnslog.NewPackageLogger("github.com/rook/rook", "cluster-manager")
+
+type ClusterManager struct {
+	OSDHostMap       map[string]string
```

Comment:
I can't find how these variables work.

Reply:
That is good pointing out. As you mentioned, these variables are currently only being updated and not used yet.
They are intended for selecting a new next attachable host in case of a compute node failure. 

Would it be better to add this code when the variables are actually used?

Reply:
> Would it be better to add this code when the variables are actually used?

Yes

Reply:
Okay, I fixed it